airbyte-integrations/connectors/source-freshdesk/source_freshdesk/manifest.yaml

```diff
@@ -0,0 +1,488 @@
+        path: "canned_response_folders/{{ stream_slice.id }}/responses"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/canned_response_folders"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "canned_responses"
+      primary_key: "id"
+
+  conversations:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_stream/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "tickets/{{ stream_slice.id }}/conversations"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/tickets"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "conversations"
+      primary_key: "id"
+
+  discussion_forums:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_stream/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "discussions/categories/{{ stream_slice.id }}/forums"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/discussion_categories"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "discussion_forums"
+      primary_key: "id"
+
+  discussion_topics:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_stream/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "discussions/forums/{{ stream_slice.id }}/topics"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/discussion_forums"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "discussion_topics"
+      primary_key: "id"
+
+  discussion_comments:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_stream/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "discussions/topics/{{ stream_slice.id }}/comments"
+      partition_router:
+        type: SubstreamPartitionRouter
+        parent_stream_configs:
+          - stream: "#/definitions/discussion_topics"
+            parent_key: id
+            partition_field: id
+    $parameters:
+      name: "discussion_comments"
+      primary_key: "id"
+
+  solution_folders:
+    type: DeclarativeStream
+    $ref: "#/definitions/base_stream"
+    retriever:
+      $ref: "#/definitions/base_stream/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "solutions/categories/{{ stream_slice.id }}/folders"
+      partition_router:
```

Comment:
Missing foreign key which [was present in previous version](https://github.com/airbytehq/airbyte/blob/bba545e7430c4251a3bcfd2fd7582041a15b05cb/airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py#L300)

Reply:
Fixed